src/graphql/seed.ts

```diff
@@ -33,17 +34,21 @@ const distort = (str: string, progress: number) =>
 
 const prisma = new PrismaClient();
 
-const seedUsers = (): Prisma.UserCreateInput[] =>
-	times(random(5, 8), () => {
+const seedUsers = async (): Promise<Prisma.UserCreateInput[]> => {
+	const passwordHash = await argon2.hash("password");
```

Comment:
Let's seed random passwords instead

Reply:
is a bit impractial though when we want to test api routes that require authorization

Reply:
Good point